File: implode/implode.py

# ATOMIC_FUNCTION:  implode (string,list) 
# TAGS: string,list
# LANG:  python
# FUNCTION: implode (string,list) 
def implode(string,l):
	return string.join(l)

start_val = ["hello","world"]
expected_end_val = "hello world"

end_val = implode(" ",start_val)

# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")

File: implode/implode.js
// ATOMIC_FUNCTION: implode (delimiter,list)
// TAGS: string,list
// LANG:  node.js
// FUNCTION: implode (delimiter,list)
function implode(delimiter,list){
  return list.join(delimiter);
}
var start_val = ["hello","world"];
var expected_end_val = "hello world"
var end_val = implode(" ",start_val);

// TEST
if(JSON.stringify(end_val)==JSON.stringify(expected_end_val)){
    console.log('PASS');
}else{
	
    console.log('FAIL');
}
